app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -332,6 +333,7 @@ class HomeFragment : Fragment(), CoroutineScope, AccountObserver {
                     return
                 }
                 invokePendingDeleteJobs()
+                if (findNavController(this).currentDestination?.id == R.id.createCollectionFragment) return
```

Comment:
Perhaps it's better to do this inside of the `showCollectionCreationFragment` function to ensure anyone calling it in any future case isn't going to run into this :)

Reply:
Good point